core/src/androidMain/kotlin/Scanner.kt

```diff
@@ -47,7 +51,16 @@ public class AndroidScanner internal constructor() : Scanner {
                 cancel("Bluetooth scan failed", ScanFailedException(errorCode))
             }
         }
-        bluetoothAdapter.bluetoothLeScanner.startScan(callback)
+
+        val scanFilter =
+            filterServices?.map { ScanFilter.Builder().setServiceUuid(ParcelUuid(it)).build() }
+                ?.toMutableList()
+        bluetoothAdapter.bluetoothLeScanner.startScan(
+            scanFilter,
+            ScanSettings.Builder().build(),
+            callback
```

Comment:
nitpick: Trailing comma, please.

```suggestion
            callback,
```